arches/app/models/migrations/10799_geojsongeometry_featureid.py

```diff
@@ -0,0 +1,195 @@
+# Generated by Django 4.2.10 on 2024-06-18 15:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "10798_jsonld_importer"),
+    ]
+
+    forward = """
+    CREATE OR REPLACE FUNCTION refresh_geojson_geometries() RETURNS BOOLEAN AS $$
+        BEGIN
+            TRUNCATE TABLE geojson_geometries;
+
+            INSERT INTO geojson_geometries(
+                tileid,
+                resourceinstanceid,
+                nodeid,
+                geom,
+                featureid
+            )
+            SELECT t.tileid,
+                t.resourceinstanceid,
+                n.nodeid,
+                ST_Force2D(
+                    ST_Transform(
+                        ST_SetSRID(
+                            (ST_Dump(
+                                st_geomfromgeojson(
+                                    (
+                                        json_array_elements(
+                                            t.tiledata::json->n.nodeid::text->'features'
+                                        )->'geometry'
+                                    )::text
+                                )
+                            )).geom,
+                            4326
+                        ),
+                        3857
+                    )
+                ) AS geom,
+                (json_array_elements(
+                    t.tiledata::json->n.nodeid::text->'features'
+                )::json->>'id')::uuid AS featureid
```

Comment:
the cast to `json` here is unnecessary, I think, as `json_array_elements` returns `json`

Reply:
change in latest